packages/blade/src/components/SideNav/docs/SideNav.stories.tsx

```diff
@@ -488,14 +515,7 @@ const SideNavExample = ({
           <SideNavItem
             as="label"
             title="Test Mode"
-            leading={
-              <Indicator
-                color={isTestModeActive ? 'notice' : 'positive'}
-                emphasis="intense"
-                accessibilityLabel=""
-              />
-            }
-            backgroundColor={isTestModeActive ? `feedback.background.notice.subtle` : undefined}
+            leading={TestIcon}
```

Comment:
Did we remove the colors completely? there was some color when test mode is enabled right?